strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactController.java

```diff
@@ -451,6 +494,116 @@ public ResponseEntity addUser(Authentication authentication)
                        .body("{\"ok\":\"user '" + authentication.getName() + "' created\"}");
     }
 
+    @DeleteMapping(path = "{storageId}/{repositoryId}/{packageScope}/{packageName}/-rev/{rev}")
+    @PreAuthorize("hasAuthority('ARTIFACTS_DELETE')")
+    public ResponseEntity unpublishPackageWithScope(@RepositoryMapping Repository repository,
+                                                    @PathVariable(name = "packageScope") String packageScope,
+                                                    @PathVariable(name = "packageName") String packageName,
+                                                    @PathVariable(name = "rev") String rev)
+    {
+
+        logger.info("Npm unpublish a package request: storageId-[{}]; repositoryId-[{}]; packageName-[{}]; revision-[{}];",
+                repository.getStorage().getId(), repository.getId(), packageName, rev);
```

Comment:
Cosmetic: Would you mind re-aligning this and having one parameter per line here? Thanks! :)